contactql/error.go

```diff
@@ -21,6 +21,7 @@ const (
 	ErrUnsupportedContains   = "unsupported_contains"   // `property` the property key
 	ErrUnsupportedComparison = "unsupported_comparison" // `property` the property key, `operator` one of =>, <, >=, <=
 	ErrUnsupportedSetCheck   = "unsupported_setcheck"   // `property` the property key, `operator` one of =, !=
+	ErrUnknownPropertyType   = "unknown_property_type"  // `type` the property type
```

Comment:
need to add support for this on RP side where we turn these into readable error messages